packages/mui-material/src/Checkbox/Checkbox.d.ts

```diff
@@ -1,16 +1,49 @@
 import * as React from 'react';
 import { SxProps } from '@mui/system';
 import { OverridableStringUnion } from '@mui/types';
-import { InternalStandardProps as StandardProps, Theme } from '..';
+import {
+  CreateSlotsAndSlotProps,
+  SlotProps,
+  InternalStandardProps as StandardProps,
+  Theme,
+} from '..';
 import { SwitchBaseProps } from '../internal/SwitchBase';
 import { CheckboxClasses } from './checkboxClasses';
 
 export interface CheckboxPropsSizeOverrides {}
 
 export interface CheckboxPropsColorOverrides {}
 
+export interface CheckboxRootSlotPropsOverrides {}
+
+export interface CheckboxSlots {
+  /**
+   * The component that renders the root slot.
+   * @default SwitchBase
+   */
+  root: React.ElementType;
+}
```

Comment:
The checkbox should have `slots.input` too. If you track back to SwitchBase, it has `SwitchBaseRoot` and `SwitchBaseInput`.

User should be able to do this:

```js
<Checkbox slotProps={{
  root: { className: '...' },
  input: { className: '...' },
}} />
```

Reply:
@siriwatknp I'm bit confused on how to handle `slots.input` here, since `input` slot is not passed to `SwitchBase` from `Checkbox` but only `inputProps` are passed.

Reply:
I see, let me check and get back to you.